src/utils/base.scss

```diff
@@ -13,3 +13,15 @@
 @import "~clk-styletest/sass/helpers";
 @import "~clk-styletest/sass/utilities/api";
 
+:root,:host{
```

Comment:
since this is masthead specific css, can this go into masthead.scss instead? 
base.css is for common styles required in all components.

Reply:
Revolved in commit [55a032e](https://github.com/GovTechSG/sgds-web-component/pull/1/commits/55a032e1a9094b352417112c81b32dc9b59f5040)